pkg/kubectl/cmd/get_test.go

```diff
@@ -162,9 +162,9 @@ func TestGetUnknownSchemaObjectListGeneric(t *testing.T) {
 		},
 		"handles second specific version": {
 			output: "unlikelyversion",
-			list:   "v1beta3",
+			list:   "v1",
 			obj1:   "unlikelyversion", // doesn't have v1beta3
-			obj2:   "v1beta3",         // version of the API response
+			obj2:   "v1",              // version of the API response
```

Comment:
Thaaat's confusing (and even moreso because I didn't see the name of the function). 

The lists are kube api versions and the objects are all non-kube objects?

The line I was referring to was:
```
apiCodec := runtime.CodecFor(api.Scheme, "v1beta3")
```